docs/configuration.md

```diff
@@ -83,3 +83,13 @@ For more information on how to use the CLI to locate a file, see the [CLI](cli.m
 ## Configuration Schema
 
 If you'd like a JSON schema to validate your configuration, one is available here: http://json.schemastore.org/prettierrc.
+
+<!--
+  ## EditorConfig
```

Comment:
I think we can add an `_available in v1.9.0+_` note just like <https://prettier.io/docs/en/options.html#prose-wrap>, so that we don't have to uncomment new feature every time and it's clearer which version is available for it.